nautobot/core/settings.py

```diff
@@ -540,6 +540,7 @@
     "django.middleware.common.CommonMiddleware",
     "django.middleware.csrf.CsrfViewMiddleware",
     "django.contrib.auth.middleware.AuthenticationMiddleware",
+    "nautobot.core.middleware.UserDefinedTimeZoneMiddleware",
```

Comment:
Is the position of this entry significant? Wondering why it's not grouped with the other Nautobot specific middleware below.